### DISCUSSION TITLE
floating point exception: in training phase, in linux

### DISCUSSION THREAD

**Discussion by yojirto** [deepmodeling/deepmd-kit#250] [Q&A]
Hello,

 In training a system about Liquid_Methanol in a linux system, I found an error saying "floating point exception (in Japanese: 浮動小数点例外). I know that this message occurs in a case that I divide some number by zero.  However, I do not know where this calculation is carried out and how to fix this problem. Please let me know how should I do?

this is a input json file I used

{
    "_comment": " model parameters",
    "model_type":	"se_r",
    "model":	{
	"type_map":		["C", "O", "H", "H"],
	"descriptor": {
	    "type":		"se_r",
	    "sel":		[46, 46, 138, 46],
	    "rcut_smth":	1.00,
	    "rcut":		6.00,
	    "neuron":		[5, 10, 20],
	    "resnet_dt":	false,
	    "seed":		1,
	    "_comment": " that's all"
	},
	"fitting_net" :{
	    "neuron":		[120, 120, 120],
	    "resnet_dt":	true,
	    "seed":		1,
	    "_comment":		"that's all"
	},
	"_comment":	" that's all"
    },
​
    "learning_rate" : {
	"start_lr":	0.005,
	"decay_steps":	5000,
	"decay_rate":	0.95,
	"_comment":	" that's all"
    },
​
    "loss" : {
	"start_pref_e":	0.02,
	"limit_pref_e":	1,
	"start_pref_f":	1000,
	"limit_pref_f":	1,
	"start_pref_v":	0,
	"limit_pref_v":	0,
	"_comment":	" that's all"
    },
​
    "_comment": " traing controls",
    "training" : {
	"systems":	["data/"],
	"set_prefix":	"set",    
        "_comment:stop_batch": 570000,
	"stop_batch":	1000000,
	"batch_size":	1,
​
	"seed":		1,
​
	"_comment": " display and restart",
	"_comment": " frequencies counted in batch",
	"disp_file":	"lcurve.out",
	"disp_freq":	100,
	"numb_test":	10,
	"save_freq":	1000,
	"save_ckpt":	"model.ckpt",
	"load_ckpt":	"model.ckpt",
	"disp_training":true,
	"time_training":true,
	"profiling":	false,
	"profiling_file":	"timeline.json",
	"_comment":	"that's all"
    },
​
    "_comment":		"that's all"

}

my tensor flow and deepmd's version is here
deepmd-kit                1.1.5               
libdeepmd                 1.2.0      
libtensorflow_cc          2.1.0           
tensorflow                1.14.0                
tensorflow-base           2.2.0           
tensorflow-estimator      2.2.0              

in other training offered in samples, I can train it successfully. Then I think something goes wrong with my input file...
